docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -2,9 +2,34 @@
 == Running the {stack} on Docker
 
 The Elastic Docker registry contains Docker images for all the products in the 
-{stack}. For more information, see:
+{stack}: https://www.docker.elastic.co/
 
-* Official Elastic Docker images: https://www.docker.elastic.co/
+To get the default distributions of {es} and {kib} up and running in Docker, you can use 
+`docker-compose`:
+
+. Pull the docker images for {es} and {kib}:
++
+["source","sh",subs="attributes"]
+--------------------------------------------
+docker pull docker.elastic.co/elasticsearch/elasticsearch:{version}
+docker pull docker.elastic.co/kibana/kibana:{version}
+--------------------------------------------
+
+. {stack-gs}/docker-compose.yml[Download the docker-compose.yml] file for the Elastic Stack:
++
+["source","yaml",subs="attributes"]
+--------------------------------------------
+include::images/docker-compose.yml[]
+--------------------------------------------
+
+. Run `docker-compose` to bring up a three node Elasticsearch cluster and Kibana:
++
+["source","sh",subs="attributes"]
+--------------------------------------------
+docker-compose up
```

Comment:
Does it make sense to document the link (http://localhost:5601) to visit Kibana and continue from there?